contracts/PreConfirmations.sol

```diff
@@ -298,25 +321,31 @@ contract PreConfCommitmentStore is Ownable {
                 txnHash,
                 bid,
                 blockNumber,
+                decayStartTimeStamp,
+                decayEndTimeStamp,
                 bHash,
                 _bytesToHexString(bidSignature)
             );
 
             address commiterAddress = commitmentDigest.recover(commitmentSignature);
 
             require(stake > (10 * bid), "Stake too low");
-
+            require(decayStartTimeStamp < decayEndTimeStamp, "Invalid decay time");
```

Comment:
Is there a scenario where `decayStartTimeStamp` should be equal to `decayEndTimeStamp`?

Reply:
I think it's unrealistic, because you'd divide by zero in this case for the range computation.

Reply:
Also we're it's not possible to have a valid decay in this scenerio